src/deserialize_sequence.cc

```diff
@@ -346,16 +346,28 @@ void load_step_setup_script(const std::filesystem::path& folder, Sequence& seque
     if (not std::filesystem::exists(folder))
         throw Error(gul14::cat("Folder does not exist: '", folder.string(), '\''));
 
+    std::string maintainers;
     std::string step_setup_script;
 
     auto stream = std::ifstream(folder / sequence_lua_filename);
     if (stream.good())
     {
         std::string line;
         while(std::getline(stream, line, '\n'))
+        {
+            if (gul14::starts_with(line, "-- maintainers:"))
```

Comment:
I like the idea of storing the maintainer in `sequence.lua`. But it reminds me – we do not yet store the sequence timeout, do we? Could we add it here as well?